install/kubernetes/helm/istio/charts/gateways/templates/autoscale.yaml

```diff
@@ -1,5 +1,5 @@
 {{- range $key, $spec := .Values }}
-{{- if and (ne $key "global") (ne $key "enabled") }}
+{{- if and (ne $key "enabled") }}
```

Comment:
this could be handled in a separate PR but I don't understand why the conditional in requirements.yaml doesn't serve this purpose?

Reply:
i will check on this, should be possible.

Reply:
@sdake re your comment:  looking through the code, I believe this is different than the condition gateway.enabled... the gateway charts is complicated where it has multiple gateways and for each you can have enabled, e.g.
```
enabled: true

istio-ingressgateway:
  enabled: true
```